### DISCUSSION TITLE
Can I use Vercel to set up domain redirection to a specific page on another domain (e.g., domain.com/path)?

### DISCUSSION THREAD

**Discussion by adrianodev97** [vercel/community#5562] [Help]
### Summary

I have a project on Vercel with a main domain.com and two additional domains within the same project. Currently, these additional domains redirect to the main domain's homepage. However, I want to redirect them to a specific page within the main domain instead of its homepage.

![2024-01-22_16-21](https://github.com/vercel/community/assets/88254828/835f2d83-cadc-4173-a00e-d30b33e5a164)

### Steps to Reproduce

Access Vercel Dashboard
Select Project
Go to Domain Settings
Configure Redirection

**Comment by amyegan** [comment]
Hi Adriano! Your framework might have a built-in way to handle routing host-specific redirects. If it's available, that's likely the best option to use. 

You can also configure [dynamic redirects](https://vercel.com/docs/edge-network/redirects#dynamic-redirects) using serverless or edge functions. Or you can use [vercel.json redirects](https://vercel.com/docs/projects/project-configuration#redirects) to send visitors to the desired path when the request matches a specific `host` value.
